infra/main.bicep

```diff
@@ -28,7 +28,7 @@ param storageContainerName string = 'content'
 param openAiServiceName string = ''
 param openAiResourceGroupName string = ''
 @description('Location for the OpenAI resource group')
-@allowed(['eastus', 'southcentralus', 'westeurope'])
+@allowed(['canadaeast', 'eastus', 'francecentral', 'japaneast', 'northcentralus'])
```

Comment:
This is the intersection of the embedding and gpt-35 regions. Unfortunately they're not available in the same regions, so we're providing less options than if we had multiple parameters.